pkg/jsonmessage/jsonmessage_test.go

```diff
@@ -23,16 +30,181 @@ func TestProgress(t *testing.T) {
 		t.Fatalf("Expected %q, got %q", expected, jp2.String())
 	}
 
+	expectedStart := "[==========>                                        ]     20 B/100 B"
+	jp3 := JSONProgress{Current: 20, Total: 100, Start: time.Now().Unix()}
+	// Just look at the start of the string
+	// (the remaining time is really hard to test -_-)
+	if jp3.String()[:len(expectedStart)] != expectedStart {
```

Comment:
Not really questioning why you're only testing the start of the string, but I am curious... why does the 50% test (next) not have the same issue?